core/modules/wsched/ChunkDisk.cc

```diff
@@ -89,7 +89,7 @@ wbase::Task::Ptr ChunkDisk::MinHeap::pop() {
 }
 
 
-void ChunkDisk::enqueue(wbase::Task::Ptr const& a) {
+void ChunkDisk::queTask(wbase::Task::Ptr const& a) {
```

Comment:
I think we would prefer to have these all spelled out with "queue" as a full word instead of the odd abbreviation "que".